ElastosDIDSDK/DIDStore.swift

```diff
@@ -12,30 +12,34 @@ public class DIDStore: NSObject {
     public static let DID_NO_PRIVATEKEY = 1
     public static let DID_ALL = 2
 
-    private var documentCache: LRUCache<DID, DIDDocument>?
-    private var credentialCache: LRUCache<DIDURL, VerifiableCredential>?
+    private var didCache: LRUCache<DID, DIDDocument>?
+    private var vcCache: LRUCache<DIDURL, VerifiableCredential>?
```

Comment:
Don't  have to follow the name with Java SDK. Swift can have  it's own naming convention.